showyourwork/config.py

```diff
@@ -313,6 +367,15 @@ def parse_config():
         #: Optimize the DAG by removing jobs upstream of cache hits
         config["optimize_caching"] = config.get("optimize_caching", False)
 
+        #: Showyourwork stamp settings
+        config["stamp"] = as_dict(config.get("stamp", {}))
+        config["stamp"]["enabled"] = config["stamp"].get("enabled", True)
+        config["stamp"]["size"] = config["stamp"].get("size", 0.75)
+        config["stamp"]["xpos"] = config["stamp"].get("xpos", 1.0)
+        config["stamp"]["ypos"] = config["stamp"].get("ypos", 1.0)
+        config["stamp"]["angle"] = config["stamp"].get("angle", -20)
+        config["stamp"]["maxlen"] = config["stamp"].get("maxlen", 30)
```

Comment:
Could the maxlen be increased to 40? For me personally I would rather have the text wrap around a little bit more before it displays a `...`.

Reply:
(My sense is that many paper slugs are longer than 30 characters - although I don't have actual stats on this...)

Reply:
40 is fine. I'll change it.